hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestContainerBalancerOperations.java

```diff
@@ -115,4 +124,47 @@ public void testContainerBalancerCLIOperations() throws Exception {
   }
 
   //TODO: add more acceptance after container balancer is fully completed
-}
+
+  /**
+   * Test if Container Balancer CLI options override the default configurations
+   */
+  @Test
```

Comment:
Thank you @sarvekshayr for adding tests, we can increase the `@Timeout` for this test class (added as part of https://github.com/apache/ozone/pull/5854) for avoiding potential intermittent failures in CI.

Reply:
Adding `@Timeout` for the new testcase or changing the existing timeout from 300 milliseconds to seconds
?

Reply:
You can modify the existing `@Timeout` for the `TestContainerBalancerOperations` test class.